app/models/curator/digital_object.rb

```diff
@@ -33,6 +34,17 @@ class DigitalObject < ApplicationRecord
       has_one :issue_for, through: :issue_mapping_for, source: :digital_object
     end
 
+    self.curator_indexable_mapper = Curator::DigitalObjectIndexer.new
+
+    def institution
+      admin_set.institution
+    end
+
```

Comment:
replace institution method to

`has_one :institution, through: :admin_set, class_name: 'Curator::Institution`  Move this Just under the `belongs_to admin_set` declaration around L12

Reply:
Fixed via 2075da6378d0901b1da558acbcc128391e49ea15